src/sentry/grouping/enhancer/actions.py

```diff
@@ -16,10 +16,10 @@
 }
 assert len(ACTIONS) < 1 << max(ACTION_BITSIZE.values())
 ACTION_FLAGS = {
-    (True, None): 0,
+    (True, ""): 0,
```

Comment:
I will make a reference to this later. This makes the type of the 2nd value of the Tuple to always be `str`